web/src/app/chat/ChatPage.tsx

```diff
@@ -137,6 +135,23 @@ export function ChatPage({
           )
         : undefined
   );
+
+  // Gather default temperature settings
+  const search_param_temperature = searchParams.get(
+    SEARCH_PARAM_NAMES.TEMPERATURE
+  );
+  const defaultTemperature = search_param_temperature
+    ? parseFloat(search_param_temperature)
+    : selectedAssistant?.tools.some(
+          (tool) => tool.in_code_tool_id === "SearchTool"
+        )
+      ? 0.7
```

Comment:
Oops, probably wasn't clear with this one but I think 0 for SearchTool (and probably websearch tool) enabled and `0.7` for without makes sense